go.mod

```diff
@@ -1,20 +1,16 @@
-module github.com/flatcar-linux/container-linux-config-transpiler
+module github.com/kinvolk/container-linux-config-transpiler
 
-go 1.15
+go 1.16
 
 require (
-	github.com/ajeddeloh/go-json v0.0.0-20160803184958-73d058cf8437 // indirect
+	github.com/ajeddeloh/go-json v0.0.0-20200220154158-5ae607161559 // indirect
 	github.com/ajeddeloh/yaml v0.0.0-20170912190910-6b94386aeefd
```

Comment:
This repo doesn't look maintained. Wonder if using a maintained fork such as https://github.com/go-yaml/yaml would make sense